src/wasm/wasm-binary.cpp

```diff
@@ -2132,9 +2132,9 @@ void WasmBinaryBuilder::readFeatures(size_t payloadLen) {
     uint8_t prefix = getInt8();
     if (prefix != BinaryConsts::FeatureUsed) {
       if (prefix == BinaryConsts::FeatureRequired) {
-        throwError("Required features not supported");
+        std::cerr << "warning: required features in feature section are ignored";
       } else if (prefix == BinaryConsts::FeatureDisallowed) {
-        throwError("Disallowed features not supported");
+        std::cerr << "warning: disallowed features in feature section are ignored";
```

Comment:
We should `continue` after printing the warning since disallowed features are not used (but required features are, so no need for a `continue` in the previous case).